cli/jobs/train/r/basic-train-model/basic_r_job.yml

```diff
@@ -0,0 +1,16 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+experiment_name: r-basic-example
```

Comment:
follow standard order of job sections (see contributing guidelines) - experiment_name should be toward bottom